engine/core/src/main/java/org/datacleaner/descriptors/ClasspathScanDescriptorProvider.java

```diff
@@ -611,7 +611,7 @@ public ClasspathScanDescriptorProvider addAnalyzerClass(Class<? extends Analyzer
             try {
                 descriptor = Descriptors.ofAnalyzer(clazz);
                 _analyzerBeanDescriptors.put(clazz.getName(), descriptor);
-            } catch (Exception e) {
+            } catch (Throwable e) {
```

Comment:
Catching throwable is something to be careful with... Difference from Exception is just errors which you cannot recover from (OOM, StackOverflow etc). Why is it needed?